addOns/replacer/src/main/java/org/zaproxy/zap/extension/replacer/ExtensionReplacer.java

```diff
@@ -136,6 +138,56 @@ private static boolean contains(String original, String match, Pattern p) {
     }
 
     private static String replace(String original, String match, Pattern p, String replacement) {
+        LOGGER.debug("Static Replacement function.");
```

Comment:
This whole replacement should be optional, as is there's no way to opt-out.

Reply:
What's the best way to implement something like that? Would an additional flag similar to the **Enabled** checkbox on each replacer item suffice?

Currently if you don't have a replacement string starting with `{{RINT`, `{{TICKS`, or `{{UUID` it will just return whatever was entered into the field.

Reply:
A flag like that makes sense.

Right but, even if it's unlikely, we shouldn't prevent the user from sending those tokens literally.

Reply:
Ok, that work has been done to add a checkbox, store the state, etc.